arches_component_lab/utils/safe_object_filter.py

```diff
@@ -0,0 +1,22 @@
+import logging
+from django.core.exceptions import FieldError
+
+logger = logging.getLogger(__name__)
+
+
+class SafeObjectFilter(object):
+
+    def __init__(self, model):
+        super().__init__()
+        self.model = model
+
+    def apply_filters(self, **kwargs):
+        object_filter = self.model.objects
```

Comment:
We can't depend on a model manager being called "objects", that's just the default. I'd probably inspect the field names (we have a [helper](https://github.com/archesproject/arches/blob/a0ad07c823f29379d529ea35872614eb3fa116e1/arches/app/models/utils.py#L17) for this)